xbmc/video/VideoDatabase.cpp

```diff
@@ -2159,20 +2160,25 @@ bool CVideoDatabase::GetFileInfo(const std::string& strFilenameAndPath, CVideoIn
     if (idFile < 0)
       return false;
 
-    std::string sql = PrepareSQL("SELECT * FROM files "
+    std::string sql = PrepareSQL("SELECT files.idFile, files.strFilename, files.dateAdded, path.strPath, "
+                                "bookmark.timeInSeconds, bookmark.totalTimeInSeconds, "
+                                "MAX(history.dateWatched), COUNT(history.idFile) FROM files "
```

Comment:
Please use aliases for aggregated columns to not repeat it below in field getter.

Reply:
Does that work? I just wasn't sure and would have changed it to the index getter on my next wip update.

Reply:
it should work and please don't use index-based getter for long select lists.